src/Skia/Avalonia.Skia/GlyphTypefaceImpl.cs

```diff
@@ -220,5 +250,65 @@ public bool TryGetStream([NotNullWhen(true)] out Stream? stream)
                 return false;
             }
         }
+
+        private static bool ReadOS2Table(byte[]? array, out int usWinAscent, out int usWinDescent)
+        {
+            if (array == null || array.Length < 78)
+            {
+                usWinAscent = 0;
+                usWinDescent = 0;
+                return false;
+            }
+
+            using (MemoryStream stream = new MemoryStream(array))
```

Comment:
Won't using Span+Slice and ReadUInt16BigEndian directly work here? Why do you need a memory stream?

Reply:
> Won't using Span+Slice and ReadUInt16BigEndian directly work here? Why do you need a memory stream?

I've rarely used Spans since I only recently started using new C# versions, so maybe that's a better solution